### PR TITLE
Release v2 for columbus-5 and bombay-9

### PR DESCRIPTION
**Pull request by hanjukim**

### CODE REVIEW

File: integration-tests/paramchangeproposal.ts [hunk 1]
```diff
@@ -38,32 +38,32 @@ const proposal = new ParameterChangeProposal('testing params', 'yay!', {
...
-    seigniorageburdentarget: new Dec('2342.234234'),
-    miningincrement: new Dec(23423423423.234234234234982),
-    windowshort: 50,
-    windowlong: 2,
-    windowprobation: 30,
+    SeigniorageBurdenTarget: new Dec('2342.234234'),
+    MiningIncrement: new Dec(23423423423.234234234234982),
+    WindowShort: 50,
+    WindowLong: 2,
+    WindowProbation: 30,
```
Comment (jarcodallo) [inline integration-tests/paramchangeproposal.ts:52]: I think these variable should start with lower case like `taxPolicy`. Usually the upper case used for Classes.

File: src/client/lcd/api/AuthAPI.spec.ts [hunk 2]
```diff
@@ -3,14 +3,15 @@ import { AuthAPI } from './AuthAPI';
 import { Account, LazyGradedVestingAccount } from '../../../core';
```
Comment (jarcodallo) [inline src/client/lcd/api/AuthAPI.spec.ts:3]: `LazyGradedVestingAccount` is defined but never used, please remove it.

File: src/client/lcd/api/TxAPI.ts [hunk 3]
```diff
@@ -318,13 +317,16 @@ export class TxAPI extends BaseAPI {
...
       const blockResult: any = {
-        height: Number.parseInt(d.height),
         txhash: d.txhash,
         raw_log: d.raw_log,
         gas_wanted: Number.parseInt(d.gas_wanted),
         gas_used: Number.parseInt(d.gas_used),
       };
 
+      if (d.height) {
+        blockResult.height = Number.parseInt(d.height);
```
Comment (jarcodallo) [inline src/client/lcd/api/TxAPI.ts:327]: The `Number` before `parseInt` is not need it. You can called like `parseInt(d.height, 10)`. Always add the radix  to it

File: src/core/market/params.ts [hunk 4]
```diff
@@ -26,18 +26,18 @@ export namespace MarketParamChange {
 
 export interface MarketParamChanges {
   market?: {
-    poolrecoveryperiod?: number;
-    basepool?: Dec;
-    minstabilityspread?: Dec;
+    PoolRecoveryPeriod?: number;
+    BasePool?: Dec;
+    MinStabilitySpread?: Dec;
```
Comment (jarcodallo) [inline src/core/market/params.ts:31]: I think these variable should start with lower case like `poolRecoveryPeriod`. Usually the upper case used for Classes.

File: src/core/oracle/params.ts [hunk 5]
```diff
@@ -52,25 +52,25 @@ export namespace OracleParamChange {
...
-    slashwindow?: number;
-    minvalidperwindow?: Dec;
+    VotePeriod?: number;
+    VoteThreshold?: Dec;
+    RewardBand?: Dec;
+    RewardDistributionWindow?: number;
+    Whitelist?: OracleWhitelist;
+    SlashFraction?: Dec;
+    SlashWindow?: number;
+    MinValidPerWindow?: Dec;
```
Comment (jarcodallo) [inline src/core/oracle/params.ts:62]: Same here I think these variable should start with lowercase.

**Review by jarcodallo — COMMENTED** [review]
These two variable (chainID and URL) are everywhere.
```js
const terra = new LCDClient({
  chainID: 'bombay-0007',
  URL: 'https://bombay-lcd.terra.dev',
});
```
It will be more cleaner to create const for chain id and URL. That way in the next update we can only change it in one place

I propose export there const from [LCDClient.ts](https://github.com/terra-money/terra.js/blob/master/src/client/lcd/LCDClient.ts) something like:
```
export const LCD_BASE_URL = 'https://lcd.terra.dev';
export const TESTNET_LCD_BASE_URL = 'https://bombay-lcd.terra.dev';
export const TESTNET_CHAIN_ID = 'bombay-0007';
export const CHAIN_ID = 'columbus-4';
```

File: integration-tests/paramchangeproposal.ts (hunk 1 above)
Comment (hanjukim) [reply integration-tests/paramchangeproposal.ts:52]: No, it is been changed

**Pull request merged**